src/repley/shadow/remote.cljs

```diff
@@ -1,10 +1,14 @@
-(ns repley.shadow.remote)
+(ns repley.shadow.remote
+  (:require [repley.config :as config]))
+
+(def default-port
+  (get-in config/default-config [:http :port]))
 
 (defn make-send [fetch]
   (fn submit
     ([value] (submit nil value))
     ([{:keys [port mode]
-       :or   {port 3001 ;; TODO: use repley.config/default-config
+       :or   {port default-port
               mode "cors"}}
       value]
      (fetch (str "http://localhost:" port "/receive")
```

Comment:
should probably get `:prefix` and `:receive-endpoint` as they constitute parts of the URL as well

Reply:
Which part of the URL is the prefix exactly?

Btw, the `merge-with merge` strategy is not going to work for strings like `:prefix`.

Reply:
oh yeah, good catch... :) 

the prefix is meant for allowing you to use some other path for the URLs, so it is the beginning.